docs/source/notebooks/table_of_contents_examples.js

```diff
@@ -64,5 +64,6 @@ Gallery.contents = {
     "MLDA_introduction": "MCMC",
     "MLDA_simple_linear_regression": "MCMC",
     "MLDA_gravity_surveying": "MCMC",
-    "MLDA_variance_reduction_linear_regression": "MCMC"
+    "MLDA_variance_reduction_linear_regression": "MCMC",
+    "GLM-hierarhical-jax": "MCMC",
```

Comment:
just a quick note to say that the mysterious pre-commit error from `Check all notebooks appear in table of contents` is due to the trailing comma, which `json` can't parse (and which, as far as I can tell, [shouldn't be used in javascript](https://stackoverflow.com/questions/5139205/javascript-can-a-comma-occur-after-the-last-set-of-values-in-an-array)) - I'll look at improving the error message if this happens
```suggestion
    "GLM-hierarhical-jax": "MCMC"
```

Reply:
Thanks!

Reply:
No worries!

> nbqa NB

What command did you run? If you run
```
pre-commit run --files docs/source/notebooks/GLM-hierarchical-jax.ipynb
```
then that _should_ be enough to pass all the (sometimes annoying, admittedly 😳 ) checks

Reply:
Oh, neat, it's obviously been too long that I did a PR :). Is this process documented somewhere?

Reply:
yup, should all be here https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style

Reply:
We could also think about enabling pre-commit CI, so that these checks are automatically implemented and committed by pre-commit when they fail

PS: what an exciting PR 🤩